flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```

Comment:
We need to validate the appdata fo apps. This file should be removed

Reply:
I know, but given recent appdata requirements most of the KDE apps do not pass so until all the appdata are fixed in upstream, we skip the check.

Reply:
I was happy to do this for existing apps in order to keep them updated, and also under the understanding that we were going to fix this upstream. For new apps I'd need to see a changeset/PR/similar to show that this is something that will be able to go away in a future release.

Reply:
I pushed some appdata improvements, but unfortunately the test build doesn't seem to test that.

Reply:
I did some appdata additions and it now seems to pass.